apps/settings/js/panels/call/panel.js

```diff
@@ -0,0 +1,466 @@
+define(function(require) {
+  'use strict';
+
+  var CallSettingsTaskScheduler =
+    require('panels/call/call_settings_task_scheduler');
+  var CallConstant = require('panels/call/call_constant');
+  var SettingsHelper = require('shared/settings_helper');
+  var SettingsPanel = require('modules/settings_panel');
+  var SettingsCache = require('modules/settings_cache');
+  var SettingsService = require('modules/settings_service');
+  var DsdsSettings = require('dsds_settings');
+  var DialogService = require('modules/dialog_service');
+
+  return function ctor_call_panel() {
+    return SettingsPanel({
+      onInit: function(panel, options) {
+        this._cardIndex = options.cardIndex || 0;
+        this._conns = window.navigator.mozMobileConnections;
+        this._conn = this._conns[this._cardIndex];
+
+        this._elements = {};
+        this._elements.panel = panel;
+        this._elements.header = panel.querySelector('gaia-header');
```

Comment:
Having so many elements to query makes me to suggest a small function like:
```
function $(selector) { return panel.querySelector(selector); }
```
and constructing `this._elements` using object literal. This helps on reducing the file size.